### ISSUE TITLE
Stutters in 0.9.17

### ISSUE THREAD

**Issue by nazarovtr** [brandonmousseau/vhvr-mod#582]
It looks like there is some camera issue.
Some frames are rendered with random 1-5 degree angular camera offset making the experience quite unpleasant. 
SteamVR performance chart is showing consistent frame time spikes of roughly 5% when these stutters happen.
Simpler scenes tend to produce less stutters. It mostly happens in bases, dungeons, middle of black forest.

I am playing on AMD RX 6800. Stream to PICO 4 using Virtual Desktop. Tried to change every available VD setting. Codecs, ASW, bitrates. Nothing helped.
Tried also a Quest 2 with Steam Link. Same issue.
Tried installing different version of AMD drivers. Did not help.

So I am kind of convinced it is VHVR bug by now.

I played VHVR a lot before Ashlands on the same setup. I did not have this issue.

**Comment by nazarovtr** [comment]
Tested 0.9.16. Same issue. Last version I played without this issue was 0.9.15. Iron Gate have updated unity in 0.217.46 which corresponds to 0.9.16. So maybe the cause is there. I do not see this bug in the flat game though. Even the mirror window does not have this issue when I have it inside the headset.

**Comment by Lordubuntu94** [comment]
https://github.com/user-attachments/assets/afda7fd5-52c8-4e44-a172-6129c26cbad9

The problem you are describing is similer to the one shown in this video?

If so, then we are experiencing the same thing...i have an AMD setup too, 5800x3d+7900xtx and tried almost everything you done with no success unfortunately

**Comment by nazarovtr** [comment]
Yes. Looks like the same issue. I did not have it on the main menu though, only in game. But looks the same.
I also have AMD CPU. 5950X.
The issue for me was actually resolved, but I do not know what exactly was the fix.

I started by updating to the latest motherboard bios. It fixed other instances of severe stuttering, but not this issue.

After that I've manually downloaded and installed latest AMD chipset drivers (with restart) and rolled back to Valheim 0.217.46 and VHVR 0.9.16.

The issue was still present.

The next day I've reapplied the memory profile in bios (bios update rolled it back).

Started 0.9.16 and the issue was not present anymore. But we could not dig the ground with pickaxes, which seems to be a known issue of 0.9.16.

In desperation I've upgraded to the latest version of the game and the mod and tried again. Everything works fine now.

So it seems that some kind of combination of BIOS update, chipset driver update, full off/on power cycle and memory frequency profile setting fixed it. There also was a big Windows update somewhere in the middle of this.

If the same kind of a ritual also helps you, let's close the bug and blame AMD. (I've regretted so many times already after buying AMD GPU, though this time it seems that the issue is with the chipset)

**Comment by Lordubuntu94** [comment]
that's odd, i'm on the latest driver version with everything...bios,chipset and gpu..guess i'll continue to try changing things until something happen

**Comment by Lordubuntu94** [comment]
ok just found out something, if i press the menu botton on the left controller and open the steam overlay/dashboard the game is perfectly fine and doesn't wobble. If i then left the overlay and return to the game it begins to wobble again. Could that be something related to steamvr rather than the mod or amd?

**Comment by nazarovtr** [comment]
It might be we are experiencing slightly different issues. When I opened SteamVR menu, the menu was stable, but the game was wobbling behind it.

**Comment by brandonmousseau** [comment]
For anyone hitting this - I haven't been able to reproduce, so haven't been able to propose any solutions. However, someone in our discord reported that they used "Oculus Debug Tool" and that disabling `Force Mipmap Generation On All Layers` fixed this for them - it may be worth trying this out. In the meantime, I'll investigate down this route.

**Comment by nazarovtr** [comment]
After the latest Virtual Desktop update the issue is back again for me. I also reinstalled all the C++ redistributables in between. So I don't know what exactly caused it. Probably redistributables somehow, since I have the issue with Steam Link too.

**Comment by EugeeneHub** [comment]
Has there been any progress regarding this issue? Im using the pico 4 with an 7900xtx and experience the same behaviour. I can also confirm that everthing looks smooth with the steam vr menu open, but as soon as i close it, everything starts to jitter when moving my head
